### DISCUSSION TITLE
One Tenant errors with Email & Exchange

### DISCUSSION THREAD

**Discussion by neweratech-au-nn** [KelvinTegelaar/CIPP#834] [Q&A]
I have found one tenant (as far I I know its the only one) that throws an 
Error when trying to view:
Exchange Mailboxes
Mailbox Client Access Settings
Message Trace (no error, but unable to trace anything)

But it works for that tenant for:
Contacts
Mailbox Statistics
Phishing Policies  (well no error, but no policies so...)

"Error 400 occured. There is an issue with the token configuration for this tenant. Please perform an access check"
{
  "status": 403,
  "data": [
    "Error 400 occured. There is an issue with the token configuration for this tenant. Please perform an access check"
  ],
  "message": "Request failed with status code 403"
}

**Comment by neweratech-au-nn** [comment]
Tenant Access Check
---domain.domain---
Succesfully connected
---domain.domain---
Failed to connect to Exchange: AADSTS50078: Presented multi-factor authentication has expired due to policies configured by your administrator, you must refresh your multi-factor authentication to access '00000002-0000-0ff1-ce00-000000000000'. Trace ID: 7a21c9d3-5ec2-4efd-b70a-de8db0383200 Correlation ID: 2349efca-bcf9-422d-8ba0-9a68f8bb867e Timestamp: 2022-03-18 04:34:03Z

But I have no MFA issues with any other domains.  But this client does/has used Microsoft MFA for the tenancy itself.

**Reply by FlightXCaptain** [reply]
Also having this issue.
